sites/friday/src/get-started/strapi/_index.md

```diff
@@ -1,4 +1,591 @@
+<--->
++++
+title=Gitlab repository
++++
+{{% guides/whitelabel/git_integration git="Gitlab" stack="Strapi" %}}
+<--->
++++
+title=Bitbucket repository
++++
+{{% guides/whitelabel/git_integration git="Bitbucket" stack="Strapi" %}}
+
+{{< /codetabs >}}
+
+## Deploy
+And just like that, it’s time to deploy!
+
+Depending on the Git workflow you choose at the beginning of this tutorial, there are two ways to deploy your source code changes.
+
+{{< codetabs >}}
+
++++
+title=Using {{% vendor/name %}} Git repository
++++
+
+When using the {{% vendor/name %}} Git repository as your main repository, you can push your code using the normal Git workflow (`git add . && git commit -m "message" && git push`) to push your source code changes to your `{{% vendor/cli %}}` remote repository, or by using {{% vendor/name %}} CLI command as seen below:
+```bash {location="Terminal"}
+{{% vendor/cli %}} push
+```
+
+<--->
++++
+title=Using third-party Git repository
++++
+
+When using an external Git repository (Github, Gitlab, or Bitbucket) to store your source code and having the Git integration feature enabled, on each code updates, you will need to use the normal Git workflow (`git add . && git commit -m "message" && git push`) to push your code to your external repository using well known Git command seen below:
+```bash {location="Terminal"}
+git push origin
+```
+
+Your Github/Gitlab/Bibucket integration process will then automatically create a new environment if you’re pushing a new Git branch and deploy changes to your corresponding environment.
+
+{{< /codetabs >}}
+
+{{% vendor/name %}} will now read your configuration files, and begin building your application image. **Your first push
+will fail**; don't worry, this is expected. At this point {{% vendor/cli %}} is not aware of the resources
+our application and service need. We need to tell it what kind of CPU, Memory, and disk to assign to the various containers. Back
+in your terminal, run:
+
+```bash {location="Terminal"}
+{{% vendor/cli %}} resources:set
+```
+
+This will launch an interactive prompt to walk you through setting up your application's resources:
+```bash {location="Terminal"}
+{{% vendor/cli %}} resources:set
+Resource configuration for the project app (123456azerty), environment main (type: production):
++----------------+---------+---------+-------------+-----------+-----------+
+| App or service | Size    | CPU     | Memory (MB) | Disk (MB) | Instances |
++----------------+---------+---------+-------------+-----------+-----------+
+| app            | not set | not set | not set     | not set   | 1         |
+| postgresql     | not set | not set | not set     | not set   | 1         |
++----------------+---------+---------+-------------+-----------+-----------+
+```
+
+We will first configure our ``app`` application and then our `postgresql` service.
+The first question is what profile size you want applied to your application image. For now let's select the minimum `0.1`:
```

Comment:
with a profile size of 0.1, the app fails to run (502 error) with an error message of:
```
FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory
```
I had to go up to _at least_ 0.25 and I'm pretty sure anything beyond the base app will need more.

Reply:
to be honest, locally, i tested it with `1` :D 
so let's say `0.25` for now?